src/models/Label.ts

```diff
@@ -1,12 +1,12 @@
-export const label = [
+export let label = [
   'none',
   'hacktoberfest',
   'good first issue',
   'bug',
   'enhancement',
   'documentation',
   'easy fix',
-] as const;
+];
```

Comment:
why'd u remove the const?

Reply:
> why'd u remove the const?

Since we have to search for custom labels, the `const` was creating a problem due to type mismatch.

![image](https://github.com/user-attachments/assets/14600f87-cb31-47b4-91be-f33ca4bedf71)

Please let me know if there is any work around, I will fix it.